services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/ios/verification/devicedata/PerDeviceDataValidator.java

```diff
@@ -61,6 +66,10 @@ public PerDeviceDataResponse validateAndStoreDeviceToken(String transactionId, S
               transactionId,
               currentTimeStamp));
       perDeviceDataResponseOptional = parsePerDeviceData(response);
+      if (perDeviceDataResponseOptional.isEmpty()) {
+        logger.warn("Received status {} with body {} and headers {} when trying to update data for iOS device.",
```

Comment:
This is not for updating but for querying
```suggestion
        logger.warn("Received status {} with body {} and headers {} when trying to query data for iOS device.",
```